specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-09-01/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1137 @@
+      "delete": {
+        "description": "Deletes a Azure Monitor PrivateLinkScope.",
+        "operationId": "PrivateLinkScopes_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateLinkScopeName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request when deleting a Azure Monitor PrivateLinkScope."
+          },
+          "204": {
+            "description": "The specified PrivateLinkScope does not exist."
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "PrivateLinkScopesDelete": {
+            "$ref": "./examples/PrivateLinkScopesDelete.json"
+          }
+        }
+      },
+      "get": {
+        "description": "Returns a Azure Monitor PrivateLinkScope.",
+        "operationId": "PrivateLinkScopes_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateLinkScopeName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An Azure Monitor PrivateLinkScope definition.",
+            "schema": {
+              "$ref": "#/definitions/AzureMonitorPrivateLinkScope"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateLinkScopeGet": {
+            "$ref": "./examples/PrivateLinkScopesGet.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
>"put [](http://example.com/codeflow?start=6&length=4)

url?

Reply:
It still belongs to the higher-level object, same as _delete_ and _get_ above: _/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/privateLinkScopes/{scopeName}_